### DISCUSSION TITLE
Table block - default empty functionality when override value formatting

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#11427] [Ideas]
Hi,

Just thought it would be worth implementing a small change to the table/table block functionality.

I have a table that displays dates, i wanted to format the dates to my regional formatting and used the configure columns -> edit cog -> value feature.

I then enter my handlebars date conversion code and everything works, except if the date value is empty. Then budibase displays the code in the UI:

![image](https://github.com/Budibase/budibase/assets/83434591/25fabb73-231e-4af8-8469-2ed4b54f1483)

Currently you can use JS or handlebars script like this, but having the budibase just display nothing if no value returned would be more ideal.

```
{{#if Value }}

{{date    Value   "DD/MM/YYYY hh:mm a" }}
{{else}}

{{/if}}
```

Thanks

**Comment by ConorWebb96** [comment]
Hey @benaunt,

Thats a strange issue what version of Budibase are you running? Are you on cloud? 

I copied your exact code into a table and it seems to work as intended.
<img width="868" alt="Screenshot 2023-08-03 at 08 00 13" src="https://github.com/Budibase/budibase/assets/126772285/ccff9bc3-3bf1-4ea7-ab07-ecfded18c66d">

I'm currently running version 2.8.33. It might be related to your Budibase/app version.

**Comment by benaunt** [comment]
Oh sorry, my code is an example to get around  the issue.

If you just use:
`{{date    Value   "DD/MM/YYYY hh:mm a" }}`

If the value is empty, the table will display the code, it would be nicer if it automatically displayed an empty cell.

Thanks

**Reply by ConorWebb96** [reply]
Ahh, that makes a lot more sense. I think your workaround is the way you should approach it anyway.

Either way I think you are right and if no value is entered it should just output as empty instead of the handlebars code.

I've created a feature request here; https://github.com/Budibase/budibase/issues/11438
